cmd/flux/create_helmrelease.go

```diff
@@ -171,6 +181,13 @@ func createHelmReleaseCmdRun(cmd *cobra.Command, args []string) error {
 		helmRelease.Spec.Values = &apiextensionsv1.JSON{Raw: json}
 	}
 
+	if hrValuesFrom.String() != "" {
```

Comment:
I think it would be better if we create a custom type like [`stringSliceValue`](https://github.com/spf13/pflag/blob/master/string_slice.go) that accepts multiple `--valuesFrom` references.

Reply:
This could come later, as it will not break the current command if we allow multiple values later on right?

Reply:
That is absolutely true, just foreseeing that once a user see this is available they expect what I describe to be possible.